src/xcode/ENA/ENA/Source/Scenes/ContactDiary/__tests__/ENAUITests_07_ContactJournalUITests.swift

```diff
@@ -142,7 +142,50 @@ class ENAUITests_07_ContactJournalUITests: XCTestCase {
 		XCTAssertEqual("PommesBude-RotWeiss", locationsTableView.cells.firstMatch.staticTexts.firstMatch.label)
 	}
 
+	func testScreenshotTwoPersonsOneLocationAndMessages() throws {
+		var screenshotCounter = 0
+		// setting up launch arguments
+		app.launchArguments.append(contentsOf: ["-diaryInfoScreenShown", "YES"])
+		app.launchArguments.append(contentsOf: ["-riskLevel", "high"])
+		
+		// navigate to desired screen
+		navigateToJournalOverview()
+		
+		// select first cell
+		app.descendants(matching: .table).firstMatch.cells.element(boundBy: 1).tap()
```

Comment:
Some shortcuts:

```
        // better: 
        app.table.firstMatch.cells.firstMatch.tap()
        // even better
        app.cells.firstMatch.Tap()
```
A perfect solution would target the cell via an `AccessibilityIdentifer`:
`app.cells["foo"].firstMatch.tap`
`firstMatch` because we might have multiple cells of that type but want to target just cells tagged as "foo".

Reply:
changed it to app.cells.element